File: services/helidon-mp-svc/src/main/java/com/perfsim/helidonmpsvc/domain/OrderItem.java
package com.perfsim.helidonmpsvc.domain;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "order_items", schema = "helidon_mp")
public class OrderItem {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;

    @Column(name = "order_id", nullable = false)
    public Long orderId;

    @Column(nullable = false, length = 64)
    public String sku;

    @Column(nullable = false)
    public Integer quantity;

    @Column(name = "price_cents", nullable = false)
    public long priceCents;
}

File: services/symfony-svc/overlay/src/Entity/OrderItem.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'order_items')]
class OrderItem
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'bigint')]
    public ?string $id = null;

    #[ORM\Column(name: 'order_id', type: 'bigint', nullable: true)]
    public ?string $orderId = null;

    #[ORM\Column(type: 'string', nullable: true)]
    public ?string $sku = null;

    #[ORM\Column(type: 'integer', nullable: true)]
    public ?int $quantity = null;

    #[ORM\Column(name: 'price_cents', type: 'bigint', nullable: true)]
    public ?string $priceCents = null;
}
